### ISSUE TITLE
EPUB audiobooks?

### ISSUE THREAD

**Issue by mattgarrish** [w3c/publ-a11y#570]
I think the EPUB metadata might be trying to too closely match ONIX in this section. An onix record can describe an audiobook, but are we really intending to say that there are audio-only EPUBs? Any EPUB has to have its audio synchronized with at least some text and relies on media overlays for playback.

Specifically, this is the check for an audio-only EPUB:

> LET all_content_audio be the result of calling [check for node](https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/#check-for-node) on package_document, /package/metadata/meta[@property="schema:accessModeSufficient" and text()="auditory"].

But any epub with text and full audio synchronization is going to have a sufficient access mode of auditory. We say to set this value in the [epub accessibility techniques](https://www.w3.org/TR/epub-a11y-tech-11/#accessModeSufficient). This is the first output check, so it will override text and audio synchronized publications, causing all synchronized books to be "audio only" (or "Audiobook with no text alternative" if the long form is used).

All we can differentiate with EPUB is a book that has full text and full audio versus a book that has partial text and full audio. I think that can be deduced from the metadata as follows:

- if synchronizedTextAudio feature is set and the publication has both auditory and textual sufficient access modes, then full text and audio are available
- if synchronizedTextAudio is set and only auditory sufficient access mode, then partial text and full audio is available

**Comment by mattgarrish** [comment]
(Side note, but I thought the "complementary audio and text" string was supposed to be removed with the change we made to rich content.)

**Comment by mattgarrish** [comment]
> Partial text and full audio makes very little sense in the case of EPUB.

Chapter headings with synchronized audio is the closest you can get to DAISY full audio. I don't expect mainstream publishers will create these, of course.

**Comment by mattgarrish** [comment]
> * full text, partial audio

Right, it's common to narrate only the body text. In that case, you'd check for synchronizedAudioText with only a textual sufficient access mode.

**Comment by clapierre** [comment]
Should be addressed in latest PR #600 at least the part of the logic with both synchronized audio and AMS containing "audio"
